bika/lims/setuphandlers.py

```diff
@@ -483,7 +482,6 @@ def addColumn(cat, col):
         addColumn(bsc, 'getCategoryTitle')
         addColumn(bsc, 'getCategoryUID')
         addColumn(bsc, 'getClientUID')
-        addColumn(bsc, 'getDepartmentTitle')
```

Comment:
See comment above, the same reasoning applies for both indexes and metadata columns.